app/build.gradle

```diff
@@ -99,13 +98,17 @@ android {
             versionNameSuffix '-DEBUG'
         }
     }
+    buildFeatures {
+        viewBinding true
+    }
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
     // Setup RG tasks
     createRaygunNotifyDeploymentTask(RAYGUN_API_TOKEN, RAYGUN_API_KEY, RAYGUN_GROUP, defaultConfig.versionName, localUserName, localUserEmail)
     createRaygunProguardTask(RAYGUN_API_TOKEN, RAYGUN_APP_PATH, RAYGUN_GROUP, defaultConfig.versionName)
+    namespace 'com.raygun.raygun4androidsample.sample'
```

Comment:
Do we need `sample` twice here?

Reply:
Huh, this is weird and I *never* realised that double-up before, I think. :rofl: 

No, we don't, but it'll need a change to the package naming in the sample app. I'll create a separate ticket for it.

Reply:
Haha, sounds like a plan!